### DISCUSSION TITLE
page naviagation

### DISCUSSION THREAD

**Discussion by iprakashlal** [getbem/getbem.github.io#283] [Q&A] [answered]
Hi BEM,

I wanted to understand how you do the page navigation on your BEM website **(http://getbem.com/introduction/)**. I wanted to know, the development of the calling different URLs without refreshing the page. 
Please help me understand this type of page navigation. @floatdrop

**Comment by floatdrop** [comment, marked as answer]
This is called [Client side routing](https://www.smashingmagazine.com/2021/06/client-side-routing-next-js/).
